src/main/java/eu/deltasource/audioplayer/player/Player.java

```diff
@@ -0,0 +1,71 @@
+package eu.deltasource.audioplayer.player;
+
+import eu.deltasource.audioplayer.playable.audioplayable.AudioPlayable;
+import eu.deltasource.audioplayer.playlist.PlayList;
+
+import java.util.List;
+
+import static eu.deltasource.audioplayer.util.MyMessages.NO_RECORDS_FOUND;
+import static eu.deltasource.audioplayer.util.MyMessages.SONG_NOT_FOUND;
+
+public abstract class Player {
+
+    protected PlayList playList;
+    protected PlayListExplorer playListExplorer;
+
+    protected Player(PlayList playList, PlayListExplorer playListExplorer) {
+        this.playList = playList;
+        this.playListExplorer = playListExplorer;
+        playListExplorer.setPlayList(this.playList);
+    }
+
+
+    public void findAllByAuthorName(String authorName) {
```

Comment:
If I read some method name "findAll...", I will expect the method to return a list of objects, but the implementation is different.
Better rename the method like `printAllSongsByAuthorName`